tests/TestCase/AuthorizationServiceTest.php

```diff
@@ -173,6 +176,33 @@ public function testApplyScopeMethodMissing()
         $result = $service->applyScope($user, 'nope', $article);
     }
 
+    public function testApplyScopeAdditionalArguments()
+    {
+        $service = new AuthorizationService(new OrmResolver());
+        $user = new IdentityDecorator($service, [
+            'id' => 9,
+            'role' => 'admin',
+        ]);
+
+        $articles = new ArticlesTable();
+        $query = $this->createMock(QueryInterface::class);
+        $query->method('getRepository')
+            ->willReturn($articles);
+
+        $query->expects($this->once())
+            ->method('where')
+            ->with([
+                'identity_id' => 9,
+                'firstArg' => 'first argument',
+                'secondArg' => false,
+            ])
+            ->willReturn($query);
+        $result = $service->applyScope($user, 'additionalArguments', $query, 'first argument', false);
```

Comment:
Will these changes also enable the use of named parameters? eg. `applyScope($user, category: $category)`

Reply:
I have added a test which proves it works 😉

Reply:
Yay!